lib/ui/text.dart

```diff
@@ -2123,7 +2123,16 @@ class ParagraphBuilder extends NativeFieldWrapperClass2 {
 Future<void> loadFontFromList(Uint8List list, {String fontFamily}) {
   return _futurize(
     (_Callback<void> callback) => _loadFontFromList(list, callback, fontFamily)
-  );
+  ).then(_sendFontChangeMessage);
+}
+
+FutureOr<void> _sendFontChangeMessage(void result) async {
+  if (window.onPlatformMessage != null)
+    window.onPlatformMessage(
+      'flutter/system',
+      utf8.encoder.convert(json.encode(<String, dynamic>{'type': 'fontsChange'})).buffer.asByteData(),
```

Comment:
This seems wasteful. Could you instead create a static buffer which already contains this data?